src/keybindings/mod.rs

```diff
@@ -4,21 +4,42 @@ mod serialization;
 #[cfg(test)]
 mod tests;
 
-pub use defaults::defaults;
-
 use std::{collections::BTreeMap, fmt, slice::Iter, str::FromStr, vec::IntoIter};
 
 use serde::Deserialize;
 use winit::event::{ModifiersState, ScanCode, VirtualKeyCode};
 
-pub type Keybindings = Vec<(Action, KeyCombo)>;
+#[derive(Debug, Clone, Deserialize, PartialEq, Eq)]
+pub struct Keybindings(Vec<(Action, KeyCombo)>);
 
 #[derive(Clone, Copy, Debug, PartialEq, Eq, PartialOrd, Ord)]
 pub enum Key {
     Resolved(VirtualKeyCode),
     ScanCode(ScanCode),
 }
 
+impl Keybindings {
+    #[inline]
```

Comment:
```suggestion
```

Nit: Slapping `#[inline]` on functions without justifying through  something like checking the asm or benchmarks is generally a code smell. Chances are LLVM will already inline this and even if it doesn't this function isn't called nearly enough to be perf sensitive